webapp/channels/src/components/admin_console/message_export_settings.tsx

```diff
@@ -51,13 +50,16 @@ const messages = defineMessages({
     exportJobStartTime_title: {id: 'admin.complianceExport.exportJobStartTime.title', defaultMessage: 'Compliance Export Time:'},
     exportJobStartTime_description: {id: 'admin.complianceExport.exportJobStartTime.description', defaultMessage: 'Set the start time of the daily scheduled compliance export job. Choose a time when fewer people are using your system. Must be a 24-hour time stamp in the form HH:MM.'},
     exportFormat_title: {id: 'admin.complianceExport.exportFormat.title', defaultMessage: 'Export Format:'},
-    exportFormat_description: {id: 'admin.complianceExport.exportFormat.description', defaultMessage: 'Format of the compliance export. Corresponds to the system that you want to import the data into.{lineBreak} {lineBreak}For Actiance XML, compliance export files are written to the exports subdirectory of the configured [Local Storage Directory]({url}). For Global Relay EML, they are emailed to the configured email address.'},
+    exportFormat_description: {
+        id: 'admin.complianceExport.exportFormatDetail.description',
+        defaultMessage: 'Format of the compliance export. Corresponds to the system that you want to import the data into.{br}{br}For Actiance XML, compliance export files are written to the exports subdirectory of the configured <a>Local Storage Directory</a>. For Global Relay EML, they are emailed to the configured email address.',
```

Comment:
also lets break it down into two texts follow https://github.com/mattermost/mattermost/pull/28438#discussion_r1791547386

Reply:
Done.